src/components/MovieDetail.tsx

```diff
@@ -0,0 +1,39 @@
+'use client';
+
+import React from 'react';
+import { FetchData } from '@/lib/types';
+import Image from 'next/image';
+
+type MovieDetailProps = {
+  movie: FetchData['results'][0];
+};
+
+export default function MovieDetail({ movie }: MovieDetailProps) {
+  if (!movie) {
+    return <div className="text-white">Loading...</div>;
+  }
+
+  return (
+    <div className="text-white">
+      <div className="relative w-full h-[415px] overflow-hidden mb-4">
+        <Image
+          src={`https://image.tmdb.org/t/p/w500${movie.poster_path}`}
+          fill
+          style={{ objectFit: 'cover' }}
+          alt={movie.title}
+          className="rounded-md"
```

Comment:
```suggestion
          alt={movie.title}
          className="rounded-md object-cover"
```

어차피 tailwindCSS를 사용하시니, 인라인 방식의 스타일링 말고 이렇게 바꿔보시는 게 좋을 것 같아요!

Reply:
아하 이렇게하면 되는군요.. 감사합니다!!!